src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -131,8 +131,8 @@ Type 'help' to get help.</value>
                   [-ConfigurationName &lt;string&gt;] [-CustomPipeName &lt;string&gt;]
                   [-EncodedCommand &lt;Base64EncodedCommand&gt;]
                   [-ExecutionPolicy &lt;ExecutionPolicy&gt;] [-InputFormat {Text | XML}]
-                  [-Interactive] [-LoadProfile] [-MTA] [-NoExit] [-NoLogo] [-NonInteractive] [-NoProfile]
-                  [-OutputFormat {Text | XML}]  [-SettingsFile &lt;filePath&gt;] [-STA] [-Version]
+                  [-Interactive] [-Login] [-MTA] [-NoExit] [-NoLogo] [-NonInteractive] [-NoProfile]
```

Comment:
This needs to be updated to reflect that `-Login` needs to be first

Reply:
What's the best way to reflect that here?

Reply:
Ignore the alphabetic order and move [-Login] to the first one.

Reply:
Ok, mindful that we use the same syntax-description syntax for both `pwsh.exe` and cmdlets, but cmdlets aren't parameter-order sensitive. OTOH, there are already (undocumented) order-sensitive parameters in `pwsh.exe`.

Reply:
That would be `-Interactive` and `-NonInteractive` (the first one is needed because of Unix, where non‑interactive is the default by convention)